pkgs/applications/misc/homely/default.nix

```
@@ -0,0 +1,26 @@
+{ stdenv, pythonPackages }:
```

And than we can list all python modules here explicit instead of pythonPackages.

---

As a concrete GitHub suggestion, 

```suggestion
{ stdenv
, buildPythonApplication
, fetchPypi
, attrs
, setuptools_scm
, requests
, simplejson
, click
, pythondaemon
}:

buildPythonApplication rec {
  version = "0.15.3";
  pname = "homely";

  src = fetchPypi {
    inherit pname version;
    sha256 = "2aa7461b583660bf8d9ad776c184fa7a1272141a5c0faaf78a76bc3e8c39db6d";
  };

  buildInputs = [ attrs setuptools_scm ];
  propagatedBuildInputs = [
    requests
    simplejson
    click
    pythondaemon
  ];
```